pages/linux/latexmk.md

```diff
@@ -2,18 +2,18 @@
 
 > Automated LaTeX document generation.
 
-- Compile filename.tex to pdf:
+- Compile a LaTeX file to PDF:
 
 `latexmk -pdf {{filename.tex}}`
 
-- Force latexmk to generate document despite errors:
+- Force `latexmk` to generate a document even if there are errors:
 
 `latexmk -f {{filename.tex}}`
 
-- Clean up current directory from files generated by latexmk except; dvi, postscript and pdf:
+- Clean up the current directory from all files generated by `latexmk` except DVI, PostScript and PDF:
```

Comment:
`from all files generated` -> `by deleting all files generated`?

Reply:
I agree the current wording is a little awkward, but I don't think avoiding it justifies making the sentence that long...

Reply:
Other sources has phrased it " Clean up all temporary files in current directory"
But i am not sure it's better, because you only know which temporary files it's referring to if you are familiar with compiling .tex files.

Reply:
I'm mainly confused about the action referred to here. Are the files deleted? Or does something else happen to them?

Reply:
Yes, they are deleted.

Reply:
Ah, ok

Reply:
So what do you propose here?

Reply:
Hrm, I'm not sure. Maybe "Cleanup and delete"? Maybe it's ok as it is.

Reply:
How about "Delete all `latexmk`-generated temporary files from the current directory"? Maybe the detail on which ones are kept goes beyond the scope of tldr.

Reply:
yeah, that sounds better! Thanks :smile_cat:

Reply:
Yeah, thats good and clear 👍